data_model.md

```diff
@@ -349,14 +349,19 @@ timestamp,value,quality_flag
 ### Uploads
 {: .anchor}
 
-Data may be uploaded to Solar Forecast Arbiter either through the API or
+Data may be uploaded to the Solar Forecast Arbiter either through the API or
 the dashboard in either CSV or JSON format. Valid files must have
 timestamps in ISO-8601 format with time zone. They must also include the
-value of the data, and a quality flag of either 0 or 1. Each file
-may only contain data for **one** observation or forecast variable.
+value of the data, and a quality flag of either 0 or 1.
+
+Each file may only contain data for **one** observation or forecast variable.
 Multiple observation or forecast variables require multiple file
 uploads to their respective dashboard pages or API end points.
 
+The Solar Forecast Arbiter does not archive missing status.
```

Comment:
So there's a pre-validation at time of upload, rather than identifying missing data using the Validator?

Reply:
The idea was that the API/dashboard will reject data uploads that look like this:

```
timestamp,value,quality_flag
2018-11-22T12:00:00Z,10.23,0
2018-11-22T12:05:00Z,NULL,0
2018-11-22T12:10:00Z,10.67,0
2018-11-22T12:15:00Z,11.00,0
```

The server returns a response like "400*: invalid data: NULL. check that the data does not contain missing data markers". The user then submits:

```
timestamp,value,quality_flag
2018-11-22T12:00:00Z,10.23,0
2018-11-22T12:10:00Z,10.67,0
2018-11-22T12:15:00Z,11.00,0
```

I'm not sure what the Validator is supposed to do with this. Maybe it depends on the application and we need to get more concrete. This makes me wonder if refusing to store data as missing means that the Validator (or parts of it) will need to be rerun every time the data is analyzed. I don't know that the Observation currently keeps enough metadata for the Validator to be able to confidently determine if data is missing. 

what am I missing @alorenzo175 ?

Reply:
I had anticipated that Validator would have a function to set a 'missing_value' flag.  OK with me if that's done by the io module.  Your example suggests that we'll allow upload of incomplete time series (i.e. gaps in the timestamps).  I think that's more problematic than storing a timestamp with a NaN.

Reply:
Forecasts must be at a regular interval, but that requirement does not apply to Observations. I expect that the majority of power data will be at irregular intervals.

Reply:
per March 7 discussion, we will require observations to be submitted with a regular interval and we will require missing data to be specified as missing.